File: register-diplomas/Diplomas.Public/Diplomas.Public.DataAccess/RoleAudit.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace Diplomas.Public.DataAccess
{
    public partial class RoleAudit
    {
        public int AuditId { get; set; }
        public int? SysUserId { get; set; }
        public string Username { get; set; }
        public int? SysRoleId { get; set; }
        public DateTime DateUtc { get; set; }
        public string Action { get; set; }
        public int? InstId { get; set; }
        public string ObjectName { get; set; }
        public int? ObjectId { get; set; }
        public int AuditModuleId { get; set; }
        public string AssignedToSysUserId { get; set; }
        public string AssignedToSysUsername { get; set; }
        public string AssignedSysRoleId { get; set; }
        public string AssignedSysRoleName { get; set; }
        public string AssignedInstitutionId { get; set; }
    }
}

File: user-management/frontend-neispuo-user-management/src/app/layout/content-layout/components/pages/update-roles-page/role-audit/role-audit.ts
import { AuditActionEnum } from '@shared/enums/audit-action.enum';

export interface RoleAudit {
    AuditId: number;
    SysUserID: number;
    Username: string;
    SysRoleId: number;
    DateUtc: Date | string;
    Action: AuditActionEnum;
    InstId: number;

    // from Data
    Data: {
        AssignedToSysUserID: number;
        AssignedToSysUsername: string;
        AssignedSysRoleID: number;
        AssignedSysRoleName: string;
        AssignedInstitutionID: number;
    };
}
